src/lem.lisp

```diff
@@ -122,6 +124,12 @@ Options:
                                      (error "Please, specify a filename to log to."))
                                    (setf (command-line-arguments-log-filename parsed-args)
                                          filename)))
+                                ((member arg '("-i" "--interface") :test #'equal)
+                                 (let ((interface (pop args)))
+                                   (unless (and interface (/=  0 (array-total-size interface)))
+                                     (error "Please, specify an interface to use."))
```

Comment:
I don't think erroring here is cool. I'd rather have a default.

Also an error throws a backtrace right? A simple message to the user might have been nicer.

Reply:
Makes sense. I've changed it to printing a string to stderr in a new commit.